app/trace/constants.ts

```diff
@@ -27,6 +27,7 @@ export const SECTION_PADDING_BOTTOM = 1;
 
 export const TRACK_HEIGHT = 16;
 export const TRACK_VERTICAL_GAP = 1;
+export const TRACK_FILTERED_OUT_COLOR = "#ddd";
```

Comment:
uber nit: EVENT_FILTERED_OUT_COLOR and move to the group below? since it applies to individual events

Reply:
Good catch, done.